src/gstvimbasrc.c

```diff
@@ -1558,6 +1559,10 @@ VmbError_t set_roi(GstVimbaSrc *vimbasrc)
                            ErrorCodeToMessage(result));
     }
     // offsetx
+    if (vimbasrc->properties.offsetx == INT_MAX) {
```

Comment:
Check for new magic number

```suggestion
    if (vimbasrc->properties.offsetx == -1) {
```